app/docs/0.13.x/proxy.md

```diff
@@ -973,13 +1022,13 @@ Server: kong/x.x.x
 ### The `http_if_terminated` property
 
 If you wish to consider the `X-Forwarded-Proto` header of your requests when
```

Comment:
The Routes doesn't seem to have an `http_if_terminated` field. I'm not sure we built an equivalent for this. Maybe this section needs to just be removed?